api/graphql/schema.graphql

```diff
@@ -124,6 +124,11 @@ type User {
   employeeProfile: EmployeeProfile @hasOne @canFind(ability: "view")
   govEmployeeProfile: BasicGovEmployeeProfile @self @canFind(ability: "view")
 
+  # Employee WFA info
+  wfaInterest: LocalizedWFAInterest @rename(attribute: "wfa_interest")
+  wfaDate: DateTime @rename(attribute: "wfa_date")
+  wfaUpdatedAt: DateTime @rename(attribute: "wfa_updated_at")
+
```

Comment:
These fields are query-able by process operators and community admins

<img width="1122" height="321" alt="image" src="https://github.com/user-attachments/assets/31530021-6f3f-42b3-819d-4594d84886a0" />

Reply:
Right, we usually put these behind a `@can` using a diffewrent type. Updated in [extract fields to sub-type of user](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14625/commits/f3ddddf484e1dffb9f2b191f2cd23ac8cd9a3c21)